.github/dependabot.yml

```diff
@@ -0,0 +1,38 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "07:00"
+      timezone: "Europe/Paris"
+    open-pull-requests-limit: 10
+    reviewers:
+      - "scality/object"
+    commit-message:
+      prefix: "gomod"
+      include: "scope"
+    labels:
+      - "gomod"
+      - "dependencies"
+
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "08:00"
+    reviewers:
+      - "scality/object"
```

Comment:
What does this do exactly?

Reply:
Does it choose random reviewers in this group? Or just assign the group whatever that means?

Reply:
I think reviewers should include scality/object squad team folks to merge. (mandatory)

Reply:
Trying it out